packages/python/plotly/plotly/express/_doc.py

```diff
@@ -82,6 +82,7 @@
         colref_desc,
         "Values from this column or array_like are used to set ids of sectors",
     ],
+    path=[colref_type, colref_desc],
```

Comment:
needs work

Reply:
hehe yes :)

Reply:
oops yes. The PR is not completely ready but I'll still appreciate your thoughts about the API and how it's implemented, then I will polish if this is the right direction, unless there is some refactoring to do first.